src/main/java/ch/njol/skript/effects/EffEquip.java

```diff
@@ -93,6 +99,7 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 
 	private static final boolean SUPPORTS_STEERABLE = Skript.classExists("org.bukkit.entity.Steerable");
 
+	private static final ItemType HELMET = Aliases.javaItemType("helmet");
 	private static final ItemType CHESTPLATE = Aliases.javaItemType("chestplate");
 	private static final ItemType LEGGINGS = Aliases.javaItemType("leggings");
 	private static final ItemType BOOTS = Aliases.javaItemType("boots");
```

Comment:
Probably wouldn't hurt to add elytra into this as a means to equip it on chestplate

Reply:
Good catch!